koku/masu/database/trino_sql/reporting_awscostentrylineitem_daily_summary.sql

```diff
@@ -76,6 +76,7 @@ SELECT uuid() as uuid,
     cast(unblended_cost * {{markup | sqlsafe}} AS decimal(24,9)) as markup_cost,
     cast(blended_cost * {{markup | sqlsafe}} AS decimal(33,15)) as markup_cost_blended,
     cast(savingsplan_effective_cost * {{markup | sqlsafe}} AS decimal(33,15)) as markup_cost_savingsplan
+    cast(calculated_amortized_cost * {{markup | sqlsafe}} AS decimal(33,9)) as markup_cost_amortized
```

Comment:
Any particular reason to use 33,9 for the decimal when we use 33,15 in most places? We're adding a column to the select but not the insert here, we probably want to insert this column into the db too.

Reply:
And theres a missing comma at the end of line 78 above